### DISCUSSION TITLE
什么时候可以支持到llama3？

### DISCUSSION THREAD

**Discussion by Zhyblx** [amithkoujalgi/ollama4j#41] [General]
什么时候可以支持到llama3？

**Comment by amithkoujalgi** [comment]
Duplicate of #40

**Comment by Zhyblx** [comment]
thank you
发自我的 iPhone

在 2024年5月14日，13:13，Amith Koujalgi ***@***.***> 写道：

﻿

Duplicate of #40<https://github.com/amithkoujalgi/ollama4j/issues/40>
